loadtest.py

```diff
@@ -1,15 +1,14 @@
 import os
+import sys; sys.path.append('.')
 
-from ailoads.fmwk import scenario
+import aiohttp
```

Comment:
I don't think this import is required (molotov will hand a aiohtttp.Client directly)